.github/workflows/build-multiarch.yml

```diff
@@ -20,8 +20,10 @@ jobs:
         variant:
           - java20
           - java20-alpine
+          - java20-oracle-graalvm
```

Comment:
Let's shorten to just `java20-graalvm` since the old ones were already qualified with "ce". If the qualifier still makes sense, then it can go at the end.